gpflow/config/__config__.py

```diff
@@ -33,10 +42,22 @@ def name(self):
 
 
 def default(value: _Values):
-    """Checks if """
+    """Checks if value is set in the environment."""
     return os.getenv(value.name, default=value.value)
 
 
+def _to_bijector(value: str) -> tfp.bijectors.Bijector:
+    bijector_map = {
+        "exp": tfp.bijectors.Exp,
+        "softplus": tfp.bijectors.Softplus,
+    }
+    if isinstance(value, str):
+        value = value.lower()
+    if value not in bijector_map:
+        raise ValueError(f"`{value}` not in set of valid bijectors: {sorted(bijector_map)}")
+    return bijector_map[value]()
```

Comment:
This already instantiates the bijector, so the config will always return the same object, which means sharing the bijector which according to @awav is actually wrong...

Reply:
Good point - I think having the actual bijectors in the config is maybe too complex - let me refactor.

Reply:
It should be fine to have the definitions in `bijectors.py` and still get the valid keys from there inside `__config__.py` without having to manually duplicate! :)

Reply:
```suggestion
    return bijector_map[value]
```
and then instantiate it inside bijectors.py's `positive()` :)